server/src/test/scala/za/co/absa/atum/server/api/controller/PartitioningControllerSpec.scala

```diff
@@ -33,9 +33,14 @@ class PartitioningControllerSpec extends ZIOSpecDefault with TestData {
 
   private val partitioningServiceMock = mock(classOf[PartitioningService])
 
-  when(partitioningServiceMock.returnAtumContext(partitioningSubmitDTO1))
-    .thenReturn(ZIO.succeed(atumContextDTO1))
-  when(partitioningServiceMock.returnAtumContext(partitioningSubmitDTO2))
+  when(partitioningServiceMock.createPartitioningIfNotExists(partitioningSubmitDTO1))
+    .thenReturn(ZIO.right(()))
+  when(partitioningServiceMock.getPartitioningMeasures(partitioningDTO1))
+    .thenReturn(ZIO.succeed(Seq(measureDTO2)))
```

Comment:
Would create a sequence of two items, just to verify than more items are handled properly

Reply:
Done.